SYCL.make

```diff
@@ -0,0 +1,11 @@
+
+sycl-stream: main.cpp SYCLStream.cpp SYCLStream.sycl
+	$(CXX) -O3 -std=c++11 -DSYCL main.cpp SYCLStream.cpp -include SYCLStream.sycl -lComputeCpp -lOpenCL -o $@
+
+
+SYCLStream.sycl: SYCLStream.cpp
+	compute++ SYCLStream.cpp -sycl -no-serial-memop -O2 -emit-llvm -c
```

Comment:
Thanks, we'll use this instead.

This doesn't include `-no-serial-memop` though (which leads to warnings from the compiler). Should we just ignore that?